Documentation editing pass (#36)
patrickpr/trapdirector: docs/15-mib.md

Database
---------------

Traps are held in the database. You can set the retention period for old traps here.

> This will occur at regular intervals (not implemented in beta) or when a trap is received.

Reviewer: ```suggestion
This will occur every time a trap is received.
```
Regular purge was planned, but as it takes very little time to execute, it was abandonned. 
If ti comes up again, I will pdate.

To drop some traps manually, you can change the number of days and click "Do it now".

Log destination
---------------

> Here is where the trap receiver will log info about what it is doing - or not doing -.

Reviewer: ```suggestion
Here is where the trap receiver will log info about what it is doing - or why he is not doing it-.
```

Author: I have manually added this with "it" pronoun.
